### DISCUSSION TITLE
魔兽大数据 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#72] [模板讨论]
魔兽大数据 模板问题反馈评论区
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 3/5 request,
Fail assert: {"re": "\"code\":[^0]", "from": "content"} from failed_asserts,
Request URL: http://bj.wowdata.top/chat/send

好像一直都是失败

**Reply by wjf0214** [reply]
更新了模板，可以再试试。应该是框架更新，cookie解析问题导致的。

**Comment by miaoxiaozhao** [comment]
Failed at 1/5 request,
Fail assert: {"re": "\\\\u672a\\\\u767b\\\\u5f55", "from": "content"} from failed_asserts,
Request URL: https://wowdata.top/quest/doQuest/signin
